00-max.js

```diff
@@ -1,3 +1,11 @@
 export function max(array) {
-    // TODO: implement max function
+    let output;
+    let isArrayHaveContent = checkArrayContent(array);
```

Comment:
You could inline (="replace the method call with the method content") this method because it's only one line and it's only used once. There's no need adding 3 lines for the method that's only used once.

```suggestion
    let isArrayHaveContent = array.length != 0 ? true :false;
```

I'm sure you can find a way to simplify it further.